src/components/views/dialogs/UseCaseSelection.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2017 Vector Creations Ltd
+Copyright 2018, 2019 New Vector Ltd
+Copyright 2019 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+
+import { _t } from "../../../languageHandler";
+import SplashPage from "../../structures/SplashPage";
+import AccessibleButton from "../elements/AccessibleButton";
+
+export enum UseCase {
+    PersonalMessaging = "PersonalMessaging",
+    WorkMessaging = "WorkMessaging",
+    CommunityMessaging = "CommunityMessaging",
+    Skip = "Skip"
+}
+
+interface Props {
+    onFinished: (useCase: UseCase) => void;
+}
+
+export default function UseCaseSelection({ onFinished }: Props) {
+    return (
+        <SplashPage className="mx_UseCaseSelection">
+            <div className="mx_UseCaseSelection_title mx_UseCaseSelection_slideIn">
+                <h1>{ _t("You’re in") }</h1>
```

Comment:
applies throughout: we replace the fancy quotes (`’`) with un-fancy ones (`'`) for reasons relating to translations and consistency, though if design says they should stay then they stay.